exemples/server/server.ts

```diff
@@ -0,0 +1,64 @@
+var express = require('express');
+var fs = require("fs");
+
+var app = express();
+
+app.get('/data', function (req, res) {
+    fs.readFile( __dirname + "/" + "daten.json", 'utf8', function (err, data) {
+        res.end( data );
+    });
+})
+
+app.get('/title/:id', function (req, res) {
+    let ID:string = req.params.id;
+    let IDNew = "ID" + ID;
+    fs.readFile( __dirname + "/" + "daten.json", 'utf8', function (err, data) {
+        let daten = JSON.parse( data );
+        let title:string;
+        let obj;
+
+        try {
+            obj = {
+                title: daten[IDNew].title,
+            }
+        }
+        catch(err) {
+            obj = "ERROR ID ist nicht vorhanden :("
+        }
+        
+        let dataString:string = JSON.stringify(obj);
+
+        res.end(dataString);
+    });
+})
+
+app.get('/board/:id', function (req, res) {
+    let ID:string = req.params.id;
+    let IDNew = "ID" + ID;
+    fs.readFile( __dirname + "/" + "daten.json", 'utf8', function (err, data) {
```

Comment:
Good for now! Looking forward to the abstraction of this into a class with methods to read and write the JSON.